commerce-theme-minium/commerce-theme-minium/src/css/_login.scss

```diff
@@ -116,6 +124,30 @@
                     margin-bottom: 0;
                 }
             }
+
+            .col-md-12.portlet-column.portlet-column-only {
```

Comment:
selectors should be sorted alphabetically -- so this should be moved up above `.navigation {`